src/lib/InstantSearch.js

```diff
@@ -249,7 +267,9 @@ class InstantSearch extends EventEmitter {
    * @return {undefined} Does not return anything
    */
   start() {
-    if (this.started) throw new Error('start() has been already called once');
+    if (this.started) {
```

Comment:
I think the new message does not inform the user that `start` has already been called as clearly. I would keep the `already been called once` at least.